python/lsst/daf/butler/cli/butler.py

```diff
@@ -79,36 +90,80 @@ def _importPlugin(pluginName):
             return None
 
     @staticmethod
-    def _getLocalCommand(commandName):
-        """Search the commands provided by this package and return the command
-        if found.
+    def _mergeCommandLists(a, b):
+        """Combine two dicts whose keys are strings (command name) and values
+        are list of string (the package(s) that provide the named command).
 
         Parameters
         ----------
-        commandName : string
-            The name of the command to search for.
+        a : dict of string to list of string
+            key is command name, value is packages that provide the command
+        b : (same as a)
 
         Returns
         -------
-        Command object
-            A Click command that can be executed.
+        dict of string to list of string
+            returns a extended with b
```

Comment:
Please use capital letters at start of sentences. Also probably should note that you are returning this for convenience since `a` has been modified.